miio/airpurifier.py

```diff
@@ -90,6 +95,11 @@ def led_brightness(self) -> Optional[LedBrightness]:
         """Brightness of the LED."""
         if self.data["led_b"] is not None:
             return LedBrightness(self.data["led_b"])
+
+        # This is the property name of the Air Purifier Pro
+        if self.data["bright"] is not None:
+            return LedBrightness(self.data["bright"])
```

Comment:
Does this mean that when the device is not air purifier pro, the "led_b" will be empty? If yes, then it's fine.

Reply:
Yep. The air purifier provides a "bright" property XOR a "led_b" property.